### ISSUE TITLE
Contract violation on install of local packages

### ISSUE THREAD

**Issue by brianeroth** [gdotdesign/elm-github-install#46]
I'm receiving the following error when trying to install an Elm package that is on my local filesystem using a relative path. Using Ruby 2.4.1 and the latest version of elm-github-install.

Any thoughts? 

```
error: Contract violation for argument 2 of 2:
        Expected: nil,
        Actual: "0.18"
        Value guarded in: ElmInstall::DirectorySource::versions
        With Contract: CollectionOf => CollectionOf
        At: PATH_TO_PROJECT/node_modules/elm-github-install/scripts/dist-1.4.0/elm-install-1.4.0-osx/lib/vendor/ruby/2.2.0/gems/elm_install-1.4.0/lib/elm_install/directory_source.rb:54 . Use --trace to view backtrace
Done
```

**Comment by gdotdesign** [comment]
This is a bug, I'll try to fix it this week. Thanks for reporting :+1:

**Comment by gdotdesign** [comment]
Fixed in v1.6.1
